plugins/trace_api_plugin/compressed_file.cpp

```diff
@@ -109,7 +109,7 @@ struct compressed_file_impl {
       }
    }
 
-   void seek( long loc, fc::cfile& file ) {
+   void seek( uint64_t loc, fc::cfile& file ) {
```

Comment:
Does it make sense to change the `long remaining = loc` to `uint64_t remaining = loc`? It's not a warning but it may be worthwhile keeping the types the same as they were before

Reply:
Yes, that is even better. Will change it to `auto remaining = loc;`